pkg/webhook/pod/pod_mutater.go

```diff
@@ -57,11 +57,12 @@ func (pc *PodAdmissionControl) mutatePod(ar *admissionv1beta1.AdmissionRequest)
 		return util.ARFail(err)
 	}
 
-	if features.DefaultFeatureGate.Enabled(features.AutoScaling) {
-		err := pc.tikvHotRegionSchedule(tc, pod)
-		if err != nil {
-			return util.ARFail(err)
-		}
+	if !features.DefaultFeatureGate.Enabled(features.AutoScaling) {
```

Comment:
Should we move this to the beginning of the function?

Reply:
SGTM

Reply:
done
I also added a new test `TestValidate`